crates/turbopack-ecmascript-plugins/src/transform/swc_ecma_transform_plugins.rs

```diff
@@ -0,0 +1,193 @@
+use anyhow::Result;
+use async_trait::async_trait;
+use swc_core::ecma::ast::Program;
+use turbo_tasks::primitives::StringVc;
+use turbo_tasks_fs::FileSystemPathVc;
+use turbopack_core::issue::{Issue, IssueSeverity, IssueSeverityVc, IssueVc};
+use turbopack_ecmascript::{CustomTransformer, TransformContext};
+
+#[turbo_tasks::value(transparent)]
+pub struct PluginModule(
```

Comment:
```suggestion
pub struct CompiledSwcPluginModule(
```

I would be a bit more specific in the naming

Reply:
I wanted to not to leak detail `compiled` part, revised struct name bit.